tests/test-git

```diff
@@ -79,7 +79,7 @@ test_no_unknown()
 }
 
 # "%s" expands to nothing if no unstaged files
-test_staged_files {
+test_staged_files () {
```

Comment:
I assume that this was just a typo, and that the change reflects your original intent.